### PR TITLE
Startup Performance Improvements

### PR DESCRIPTION
**Pull request by rmadsen-ks**
- SimpleTapAssemblyResolver is no longer needed as we dont deploy self-contained installations.
- Added minor plugin searcher performance improvements

### CODE REVIEW

**Review by alnlarsen — CHANGES_REQUESTED** [review]
This is a nice cleanup, but it looks like it breaks on MacOS / Linux

**Review by alnlarsen — APPROVED** [review]
This doesn't seem to have a measurable impact on performance, but it's a nice cleanup. Having a throwaway assembly resolver is an insane way to load OpenTap.dll.

File: Cli/TapInitializer.cs [hunk 1]
```diff
@@ -26,63 +25,37 @@ public void EventsLogged(IEnumerable<Event> events)
...
+            public static readonly InitTraceListener Instance = new InitTraceListener();  
+        }
+
+        static IEnumerable<string> OpenTapLocations
+        {
+            get
+            {
+                var loc = Path.GetDirectoryName(Assembly.GetEntryAssembly().Location);
+                yield return Path.Combine(loc, "OpenTap.dll");
+            }
```
Comment (AsgerIversen) [inline Cli/TapInitializer.cs:37]: Isn't there always only one item in this enumerable?

**Review by AsgerIversen — CHANGES_REQUESTED** [review]
One small comment. Otherwise looks good.

File: Cli/TapInitializer.cs (hunk 1 above)
Comment (rmadsen-ks) [reply Cli/TapInitializer.cs]: I've simplified it.

**Review by AsgerIversen — APPROVED** [review]

**Pull request merged**